website/src/components/flex/Flex.js

```diff
@@ -37,6 +34,12 @@ const Flex = ({ sections }) => {
     // image loading for first sections, etc.
     section.index === i;
 
+    // look for any passed in classes, pass along to child
+    section.customClasses = section.sectionClasses?.split(" ");
```

Comment:
I'm not sure the `?` will help in this scenario - neither `null.split(' ')` nor `undefined.split(' ')` would be valid. Might be better to `section.sectionClasses && ...`